packages/medusa/src/types/global.ts

```diff
@@ -100,10 +100,19 @@ export type ModuleExports = {
   models?: Constructor<any>[]
 }
 
+type SessionOptions = {
+  resave?: boolean
+  saveUninitialized?: boolean
+  secret?: string
+  ttl?: number
+}
+
 export type ConfigModule = {
```

Comment:
self-review: At some point, we should change the config to follow our casing convention. 

Not super pretty to have `projectConfig > some_config > someOtherConfig`